config/locales/fr.yml

```diff
@@ -81,10 +81,10 @@ fr:
           notification_title: La pétition <a href="%{resource_path}">%{resource_title}</a> a reçu une réponse.
       proposals:
         author_confirmation_proposal_event:
-          email_intro: 'Votre proposition "%{resource_title}" a été reçue avec succès et est maintenant publique. Merci pour votre participation! Vous pouvez la consulter ici : <a href="%{resource_path}">%{resource_title}</a>.'
-          email_outro: Vous recevez cette notification car vous êtes l’auteur de la proposition. Vous pouvez vous désabonner en visitant la page de la proposition ("%{resource_title}") et en cliquant sur "Ne plus suivre".
+          email_intro: 'Votre proposition « %{resource_title} » a été reçue avec succès et est maintenant publique. Merci pour votre participation ! Vous pouvez la consulter ici :'
+          email_outro: Vous recevez cette notification car vous êtes l’auteur de la proposition. Vous pouvez vous désabonner en visitant la page de la proposition (« %{resource_title} ») et en cliquant sur « Ne plus suivre ».
           email_subject: Votre proposition a été publiée !
-          notification_title: Votre proposition <a href="%{resource_path}">%{resource_title}</a> est maintenant en ligne.
```

Comment:
You should keep the removed ' " ' char otherwise "a" HTML tag won't be valid

`<a href="%{resource_path}">`

Reply:
<img width="872" alt="Capture d’écran 2024-11-14 à 15 44 44" src="https://github.com/user-attachments/assets/663ee6b5-034e-4d38-a374-7e4d5a3d01b9">